index.html

```diff
@@ -1498,6 +1515,60 @@
 
+## Support Choosing Which Information to Present {#support-choosing-info}
+
+<div class="practice">
+  <span class="practicelab" id="principle-support-choosing-info">
+    [=User agents=] should support [=people=] in choosing which information they provide to [=parties=] that
+    request it, up to and including allowing for the provision of information that may not be factually
+    correct.
+  </span>
+</div>
+
+[=Parties=] can invest time and energy into automating ways of gathering [=data=] from [=people=] and can
+design their products in ways that make it a lot easier for [=people=] to disclose information than not, whereas
+[=people=] typically have to manually wade through options, repeated prompts, and deceptive patterns. In many
+cases, the absence of data — when a [=person=] refuses to provide some information — can also be identifying
+or revealing. Additionally, APIs can be defined or implemented in rigid ways that can prevent people from
+accessing useful functionality (eg. if my geolocation is forcefully set to match my GPS, a restaurant-finding
+site might restrict me to my current location when I want to look for restaurants in a city I will be visiting
+this weekend).
+
+[=User agents=] should make it simple for [=people=] to present the identity they wish to and to provide information
+about themselves or their devices in ways that they control. This empowers [=people=] to live in obscurity
+([[?Lost-In-Crowd]], [[?Obscurity-By-Design]]) by rendering the provision of information according to their own
+preferences as easy as possible ([[?Obfuscation]]).
+
+<div class="practice">
+  <span class="practicelab" id="principle-support-choosing-info">
+    APIs should be designed such that data returned through an API does not assert a fact or make a
+    promise on the user's behalf about the user or their environment.
+  </span>
+</div>
+
+Instead, the API could indicate a [=person=]'s preference, a [=person=]'s chosen identity, a
+[=person=]'s query or interest, or a [=person=]'s selected communication style. 
+
+In any rare instances when an API must be defined as returning true current values, users may still
+configure their agents to respond with other information, for reasons including testing, auditing or
+mitigating forms of data collection, including <a>browser fingerprinting</a>. In some cases, sites
+do not abide by data minimisation principles and request more information than they require. This
+principle supports [=people=] in defending under such cases.
+
+Sites should include deception in their threat modeling and not assume that Web platform APIs
+provide any guarantees of consistency, currency or correctness about the user. People often have
+control of the devices and software they use to interact with web sites. In response to site
+requests, people may provide information that may be deceptive for a variety of reasons, including
+both malice and self-protection.
```

Comment:
We'll reverse these two paragraphs so that the "users may still configure their agents" bit is clearly advice for site owners rather than a statement about how all APIs actually work.